### PR TITLE
MISUV-8691 Add new feature switch for reporting frequency page

### PR DESCRIPTION
**Pull request by mywyau**
App config changes
Base -> https://github.com/hmrc/app-config-base/pull/11217/files
QA -> https://github.com/hmrc/app-config-qa/pull/23803/files
Staging -> https://github.com/hmrc/app-config-staging/pull/16230
Prod -> https://github.com/hmrc/app-config-production/pull/23762

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -174,6 +174,8 @@ feature-switch {
   enable-time-machine = false
   enable-session-data-storage = true
   enable-review-and-reconcile-poa = true
+  enable-review-and-reconcile-poa = true
```
Comment (danielmaltz1) [inline conf/application.conf:177]: You've duplicated a line here

File: app/controllers/ReportingFrequencyPageController.scala [hunk 2]
```diff
@@ -16,45 +16,51 @@
...
-        errorHandler(isAgent).showInternalServerError()
-    }
-  }
-
-  def show(isAgent: Boolean = false): Action[AnyContent] = auth.authenticatedAction(isAgent) {
-    implicit user =>
-      withRecover(isAgent) {
-        optInService.setupSessionData().map(_ => Ok(s"Reporting Frequency Page. Time:${LocalDateTime.now()}"))
+  def show(isAgent: Boolean = false): Action[AnyContent] =
+    auth.authenticatedAction(isAgent) { implicit user =>
```
Comment (avelx) [inline app/controllers/ReportingFrequencyPageController.scala:50]: I think we already have different approach for agent Auth action in place, if we can use new approach instead if possible (there were some changes pushed last Friday by EMA team)

File: app/models/admin/FeatureSwitchName.scala [hunk 3]
```diff
@@ -82,35 +90,52 @@ object FeatureSwitchName {
...
-    Set(ITSASubmissionIntegration, ChargeHistory, NavBarFs, CreditsRefundsRepay,
-      PaymentHistoryRefunds, IncomeSourcesNewJourney, IncomeSources, OptOut,
-      AdjustPaymentsOnAccount, ReviewAndReconcilePoa, FilterCodedOutPoas)
+    Set(
+      ITSASubmissionIntegration,
+      ChargeHistory,
+      NavBarFs,
+      CreditsRefundsRepay,
+      PaymentHistoryRefunds,
+
```
Comment (Mbrookesy) [inline app/models/admin/FeatureSwitchName.scala:99]: There's a gap here

Comment (mywyau) [reply app/models/admin/FeatureSwitchName.scala]: done

**Review by Mbrookesy — APPROVED** [review]

**Pull request merged**